spec/javascripts/components/leaflet_iiif_spec.js.jsx

```diff
@@ -0,0 +1,13 @@
+/* globals LeafletIiif */
+
+describe('LeafletIiif', () => {
+  it('sets up the iiif layer', () => {
+    const component = TestUtils.renderIntoDocument(
+      <LeafletIiif
+        iiifImage="https://stacks.stanford.edu/image/iiif/hg676jb4964%2F0380_796-44/info.json"
+      />,
+    );
```

Comment:
I'm not really sure how to fix this, if I remove the trailing comma, eslint complains about something else. /shrug any suggestions?